qubes/vm/qubesvm.py

```diff
@@ -187,6 +187,25 @@ class QubesVM(qubes.vm.mix.net.NetVMMixin, qubes.vm.BaseVM):
             :param event: Event name (``'domain-start'``)
 
             *other arguments are as in :py:meth:`start`*
+
+        .. event:: domain-paused (subject, event)
+
+            Fired when the domain has been paused.
+
+            Handler for this event can be asynchronous (a coroutine).
```

Comment:
I'm not actually sure this is true, I just suspect it is given similar events.

I fire it from a libvirt event handler callback. Throwing from an event handler is not meaningful,as there's nothing to prevent. It's just a notification.

Reply:
Noted, thanks for the review :)